db/seeds.rb

```diff
@@ -47,7 +47,23 @@
 # end
 
 
-# Practicing with looping through arrays with each.do
-topic_list.zip(source_list).each do |topic, source|
-  LearningMaterial.create(topic: topic, source: source, description: FFaker::BaconIpsum.sentence, level: level_list.sample)
+# Practicing seeding with arrays #2
+# topic_list.zip(source_list).each do |topic, source|
+#   LearningMaterial.create(topic: topic, source: source, description: FFaker::BaconIpsum.sentence, level: level_list.sample)
+# end
+
+# Practicing seeding with arrays #3
+
+topics = topic_list.each do |topic|
+  LearningMaterial.create(topic: topic)
+end
+
+learning_materials = LearningMaterial.unscoped.all
```

Comment:
@F3PiX does this look like the exercise you had in mind? do you have a better solution for selecting the learning materials than with LearningMaterial.all?